### PR TITLE
Uninstall module used on PS 1.6 before using this one

### PR DESCRIPTION
**Pull request by Quetzacoalt91**

### CODE REVIEW

File: ps_brandlist.php [hunk 1]
```diff
@@ -62,8 +67,17 @@ public function __construct()
 
     public function install()
     {
+        if ($this->uninstallPrestaShop16Module()) {
+            // PS 1.6 was installed. Migrate and clean
+            Configuration::updateValue('BRAND_DISPLAY_TEXT_NB', Configuration::get('MANUFACTURER_DISPLAY_TEXT_NB', null, null, null, 5));
+
+            Configuration::deleteByName('MANUFACTURER_DISPLAY_TEXT');
+			Configuration::deleteByName('MANUFACTURER_DISPLAY_TEXT_NB');
```
Comment (PierreRambaud) [inline ps_brandlist.php]:
```suggestion
            Configuration::deleteByName('MANUFACTURER_DISPLAY_TEXT_NB');
```

File: ps_brandlist.php [hunk 2]
```diff
@@ -62,8 +67,17 @@ public function __construct()
...
+			Configuration::deleteByName('MANUFACTURER_DISPLAY_FORM');
```
Comment (PierreRambaud) [inline ps_brandlist.php]:
```suggestion
            Configuration::deleteByName('MANUFACTURER_DISPLAY_FORM');
```

File: ps_brandlist.php (hunk 1 above)
Comment (Quetzacoalt91) [reply ps_brandlist.php]: Weird, thanks

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by sarahdib** [comment]
@Quetzacoalt91 the old module is still install and it wasn't replace by the new one.

**Comment by matks** [comment]
@Quetzacoalt91 As I explained [in this issue](https://github.com/PrestaShop/PrestaShop/issues/26973) I will clean our open PRs backlog and close these stale PRs. But I hope that one day we get some bandwidth to reopen this topic, and when it happens, we'll reopen these PRs, find the issues noticed by QA, fix them and merge them ❤️  .

**Pull request closed without merging**